pkgs/top-level/all-packages.nix

```diff
@@ -26248,6 +26248,8 @@ in
 
   armagetronad = callPackage ../games/armagetronad { };
 
+  armagetronad-dedicated = callPackage ../games/armagetronad { dedicatedServer = true; };
```

Comment:
```suggestion
  armagetronad-server = callPackage ../games/armagetronad { dedicatedServer = true; };
```

or dedicated server which is a bit long.

Reply:
Hmm, the compiled binary is called armagetronad-dedicated. I was going for consistency with that.

Reply:
Then it makes sense to keep the name.